docs/how-to/use_elastic_store.md

```diff
@@ -0,0 +1,61 @@
+# How-To: Use Elastic to Store Similarity Index
+
+[The similarity index](../concepts/similarity_indexes.md) is a feature provided by db-ally designed to map user input to the closest matching value in a data source. In this guide, we will show you how to execute similarity search by using elastic search.
```

Comment:
Let's focus here on the ElasticStore class:

1. Add ElasticStore to `docs/reference/similarity/similarity_store/elastic.md`
2. Here we can write that `ElasticStore[link to reference]` can be used as a store in `SimilarityIndex[link to similarity index concept]` in similiar manner that we have in chroma doc